### ISSUE TITLE
Error on try to use BatchUpdate with null value

### ISSUE THREAD

**Issue by igorbmaciel** [borisdj/EFCore.BulkExtensions#156]
I tried to update a column value to a null value using BatchUpdate, but a got a error message:
"SET Columns not defined. If one or more columns should be updated to theirs default value use 'updateColumns' argument."

My Code:

Context.ReservationItems
                 .AsNoTracking()
                 .Where(r => r.RoomId.Equals(roomId) && r.TenantId.Equals(_applicationUser.TenantId) && (
                             r.ReservationItemStatusId.Equals((int)ReservationStatus.ToConfirm) ||                            
                             r.ReservationItemStatusId.Equals((int)ReservationStatus.Confirmed)))
                             .IgnoreQueryFilters()
                 .BatchUpdate(new ReservationItem { RoomId = null });

**Comment by borisdj** [comment]
The exception said it:
_"If one or more columns should be updated to theirs default value use 'updateColumns' argument."_
ReadMe has example how it is used, and in this case it would be:
```C#
var updateColumns = new List<string> { nameof(ReservationItem.RoomId) };
.BatchUpdate(new ReservationItem { RoomId = null }, updateColumns);
```
But I have found another bug when setting to Null value, which is now fixed with latest version 2.4.1 so update the NuGet as well.

**Comment by igorbmaciel** [comment]
> The exception said it:
> _"If one or more columns should be updated to theirs default value use 'updateColumns' argument."_
> ReadMe has example how it is used, and in this case it would be:
> 
> ```cs
> var updateColumns = new List<string> { nameof(ReservationItem.RoomId) };
> .BatchUpdate(new ReservationItem { RoomId = null }, updateColumns);
> ```
> 
> But I have found another bug when setting to Null value, which is now fixed with latest version 2.4.1 so update the NuGet as well.

Thanks for the answer. I tried to use default values, but not work as well.
I'll try this one now there is fixed.
